safe-api-koa/src/index.js

```diff
@@ -30,6 +32,17 @@ const serverDataVisitor: SafeAPI.Visitor<ServerDataF> = {
       ...data,
       queryParams: { ...data.queryParams, ...queryParams }
     };
+  },
+  handleCaptureParam: captureParam => data => {
+    const paramName = Object.keys(captureParam)[0];
```

Comment:
Can we throw a runtime error that the object needs to be a singleton object? And then add two unit tests that we get runtime errors when called with 1. no fields, and 2. more than two fields.

Reply:
(ignore, didn't see the runtime check at the constructor)